packages/cadl-ranch-dashboard/src/components/tree-table/row-label-cell.tsx

```diff
@@ -98,3 +99,21 @@ const GotoSourceButton: FunctionComponent<ShowSourceButtonProps> = ({ scenario }
 function getGithubLineNumber(value: number): `L${number}` {
   return `L${value + 1}`;
 }
+
+function getLabelForRow(row: TreeTableRow): string {
+  const countLeafChildren = (node: ManifestTreeNode): number => {
+    if (Object.keys(node.children).length === 0) {
+      return 1;
+    }
+    return Object.values(node.children).reduce((acc, child) => acc + countLeafChildren(child), 0);
+  };
+
+  const { name } = row.item;
+
+  if (!row.hasChildren) {
+    return name;
+  }
+
+  const totalLeafChildren = countLeafChildren(row.item);
```

Comment:
This might be a bit costly to do on every rerender and freeze the UI. Should maybe use a `useMemo`

Reply:
@timotheeguerin done